pkg/apis/internalversion/zz_generated.conversion.go

```diff
@@ -243,6 +244,7 @@ func autoConvert_v1alpha1_KwokctlConfigurationOptions_To_internalversion_Kwokctl
 	out.EtcdVersion = in.EtcdVersion
 	out.PrometheusVersion = in.PrometheusVersion
 	out.DockerComposeVersion = in.DockerComposeVersion
+	// WARNING: in.KindVersion requires manual conversion: does not exist in peer-type
```

Comment:
Why is there no KindVersion

Reply:
I found that KindVersion is unused. But I will add it